src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -133,36 +152,64 @@ private IEnumerable<PlotMapping> retrieveChartDefinitionsForSimulation(Simulatio
          new SimulationMapping
          {
-            Path = FileHelper.CreateRelativePath(simulationExport.SimulationFolder, reportFile),
+            Path = relativePath(simulationExport.SimulationFolder, configuration.ReportConfigurationFile),
             RefProject = simulationExport.ProjectName,
             RefSimulation = simulationExport.SimulationName
          };
 
-      private ObservedDataMapping[] exportObservedData(PKSimProject project, QualifcationConfiguration qualifcationConfiguration)
+      private ObservedDataMapping[] exportObservedData(PKSimProject project, QualifcationConfiguration configuration)
       {
          if (!project.AllObservedData.Any())
             return null;
 
-         var observedDataOutputFolder = qualifcationConfiguration.ObservedDataFolder;
+         var observedDataOutputFolder = configuration.ObservedDataFolder;
          DirectoryHelper.CreateDirectory(observedDataOutputFolder);
 
          return project.AllObservedData.Select(obs =>
          {
             var fileFullPath = Path.Combine(observedDataOutputFolder, $"{FileHelper.RemoveIllegalCharactersFrom(obs.Name)}{Constants.Filter.XLSX_EXTENSION}");
-            var relativePath = FileHelper.CreateRelativePath(fileFullPath, qualifcationConfiguration.ReportConfigurationFile);
             _logger.AddDebug($"Observed data '{obs.Name}' exported to '{fileFullPath}'", project.Name);
             _dataRepositoryTask.ExportToExcel(obs, fileFullPath, launchExcel: false);
             return new ObservedDataMapping
             {
                Id = obs.Name,
-               Path = relativePath,
+               Path = relativePath(fileFullPath, configuration.ReportConfigurationFile)
             };
          }).ToArray();
       }
 
+      private Task<InputMapping[]> exportInputs(PKSimProject project, QualifcationConfiguration configuration)
+      {
+         if (configuration.Inputs == null)
+            return Task.FromResult<InputMapping[]>(null);
+
+         return Task.WhenAll(configuration.Inputs.Select(x => exportInput(project, configuration, x)));
+      }
+
+      private async Task<InputMapping> exportInput(PKSimProject project, QualifcationConfiguration configuration, Input input)
+      {
+         var buildingBlock = project.BuildingBlockByName(input.Name, input.Type);
+
+         var inputsFolder = configuration.InputsFolder;
+         var projectName = FileHelper.RemoveIllegalCharactersFrom(project.Name);
+         var buildingBlockName = FileHelper.RemoveIllegalCharactersFrom(input.Name);
+         var targetFolder = Path.Combine(inputsFolder, projectName, input.Type.ToString());
+         DirectoryHelper.CreateDirectory(targetFolder);
+
+         var fileFullPath = Path.Combine(targetFolder, $"{buildingBlockName}{CoreConstants.Filter.MARDOWN_EXTENSION}");
+         await _markdownReporterTask.ExportToMarkdown(buildingBlock, fileFullPath);
```

Comment:
This is a dummy implementation of the markdwon reporter for now until we know more about what we need!